packages/flutter_app/pubspec.yaml

```
@@ -41,7 +41,7 @@ dependencies:
   package_info_plus: ^3.0.2
   rest_api_client:
     path: ../rest_api_client
-  riverpod_annotation: ^2.0.0
+  riverpod_annotation: ^2.1.0
```

こちらも🙏
https://pub.dev/packages/riverpod_annotation/changelog

```suggestion
  riverpod_annotation: ^2.1.1
```